client/src/partials/patients/documents/modals/documents.modal.html

```diff
@@ -0,0 +1,70 @@
+<div class="modal-header">
+  <span class="glyphicon glyphicon-paperclip"></span>
+  {{ 'PATIENT_DOCUMENT.ADD' | translate }} {{ 'PATIENT_DOCUMENT.DOCUMENTS' | translate }}
+  <span class="label label-primary"> {{ $ctrl.patient.abbr + $ctrl.patient.reference }}</span>
+</div>
+
+<div class="modal-body">
+  <form class="form" name="documentForm" novalidate>
+
+    <div class="form-group">
+      <label class="require" for="title">{{ 'FORM.LABELS.TITLE' | translate }}</label>
+      <input
+        class="form-control"
+        type="text"
+        name="title"
+        ng-model="$ctrl.title"
+        required>
+    </div>
+
+    <!-- thumbnail  -->
+    <img class="document-upload-thumbnail" ngf-src="$ctrl.file || 'assets/placeholder.gif'">
+
+    <!-- progress bar  -->
+    <div class="progress" ng-show="$ctrl.file.progress >= 0">
+      <div class="progress-bar progress-bar-success progress-bar-striped active"
+        style="width:{{$ctrl.file.progress}}%"
+        ng-bind="$ctrl.file.progress + '%'"></div>
+    </div>
+
+    <!-- dismiss button  -->
+    <div ng-show="$ctrl.file" class="form-group">
+      <button
+        class="btn btn-danger btn-sm"
+        ng-click="$ctrl.file = null"
```

Comment:
It is best to keep logic out of the template.  Could you move this into a function in the controller?